src/MiniJuvix/Internal/Strings.hs

```diff
@@ -217,3 +217,6 @@ minicRuntime = "minic-runtime.h"
 
 putStrLn_ :: IsString s => s
 putStrLn_ = "putStrLn"
+
+juvixFunctionT :: IsString s => s
```

Comment:
For naming consistency, consider to change this to `miniJuvixFunctionT`? similarly, for the name below.

Reply:
thanks - I'll fix this